docs/templates/datasets.md

```diff
@@ -56,8 +56,7 @@ from keras.datasets import imdb
 (X_train, y_train), (X_test, y_test) = imdb.load_data(path="imdb.pkl",
                                                       nb_words=None,
                                                       skip_top=0,
-                                                      maxlen=None,
-                                                      test_split=0.1)
+                                                      maxlen=None)
```

Comment:
The 3 last keyword arguments should appear here (mostly to make the default values explicit).